backend/src/charityprojects/models.py

```diff
@@ -70,7 +70,7 @@ def __str__(self):
 
 class LearnNewSkill(models.Model):
     objects = None
-    new_skill = models.CharField(max_length=255)
+    new_skill = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Is it not mandatory to have the skill for a particular record to get saved? It should be a required field.